common/lib/capa/capa/capa_problem.py

```diff
@@ -814,6 +814,10 @@ def _extract_html(self, problemtree):  # private
 
         if tree.tag in html_transforms:
             tree.tag = html_transforms[problemtree.tag]['tag']
+            if problemtree.tag == 'problem':
+                tree.set('class', 'questions')
```

Comment:
@cptvitamin I have decided to remove this div altogether. We already have a div with class [problem](https://github.com/edx/edx-platform/blob/master/lms/templates/problem.html#L10). Also after removing div with aria-live, the final HTML structure will looks like below

```HTML
<div class="problem">
    <div class="question">
    </div>
    <div class="question">
    </div>
   ...
```